content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,214 @@
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the prepaid services
+product configured in the steps above. Update the :guilabel:`Quantity` field with the number of
+hours. After updating any other necessary information, :guilabel:`Confirm` the quotation. Then
+click :guilabel:`Create Invoice`.
+
+Create and send an invoice for prepaid services
+-----------------------------------------------
+
+This will open a :guilabel:`Create Invoices` pop-up window. If a :guilabel:`Down Payment` needs
+to be collected, change the :guilabel:`Create Invoice` type. Then click :guilabel:`Create Invoice`.
+The invoice can now be sent to the customer for payment.
+
+Create helpdesk ticket for prepaid services
+-------------------------------------------
+
+Next, navigate to the :menuselection:`Helpdesk` app and select a :guilabel:`Team`. On the
+:guilabel:`Pipeline`, click :guilabel:`New` to create a new ticket.
+
+Fill out the ticket information, and enter the :guilabel:`Customer` information. When the customer
+name is added, the :guilabel:`Sales Order Item` field will automatically populate with the most
+recent prepaid sales order item that has time remaining.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Click on the :guilabel:`Timesheets` tab on the :guilabel:`Ticket`. To track the time spent on this
+ticket against the hours sold on the :abbr:`SO (sales order)`, click :guilabel:`Add a line`. Choose
+an :guilabel:`Employee`, and enter the number of :guilabel:`Hours Spent`. As new lines are added to
+tab, the :guilabel:`Remaining Hours on SO` is updated.
+
+.. image:: track_and_bill/track-bill-remaining-hours-total.png
+   :align: center
+   :alt: View of the timesheets tab on a ticket with an emphasis on the remaining hours on an SO.
+
+.. note::
+   If the number of hours on the :guilabel:`Timesheets` tab goes over the number of hours sold, the
+   :guilabel:`Remaining Hours of SO` will turn red.
+
+As hours are added to the :guilabel:`Timesheets` tab, they are updated in the :guilabel:`Delivered`
+field on the :guilabel:`Sales Order`.
+
+Invoice post-paid support services
+==================================
+
+When support services are billed based on the amount of time spent on an issue, an invoice cannot
+the total number of hours required to solve the problem have been entered on a timesheet. In this
+case, a service product with the invoicing policy set to :guilabel:`Based on Timesheets` (like the
+one created above) would be used.
+
+Create a sales order with a time tracked product
+------------------------------------------------
+
+To invoice a customer for post-paid support services, first create a sales order (SO) with the
+support services product. To do this, go to :guilabel:`Odoo Dashboard --> Sales --> Orders -->
+Quotations --> New`.
+
+Fill out the :guilabel:`Quotation` with the :guilabel:`Customer` information.
```

Comment:
Then, proceed to fill out the :abbr:`RfQ (Request for Quotation)` form with the necessary :guilabel:`Customer` information.